client/js/screens/Event/Event.js

```diff
@@ -1,9 +1,167 @@
 import React from 'react';
-import {Text} from 'react-native';
+import {Text, View, Image, TextInput, ScrollView} from 'react-native';
+import eventImage from '../../assets/Imagery/ManAndTree.png';
+import commentIcon from '../../assets/miscicons/question.png';
+import styles from './styles';
+import {TouchableOpacity} from 'react-native-gesture-handler';
+import Comment from '../../components/Comment';
+import dot from '../../assets/artwork/blackspot.png';
+import * as AddCalendarEvent from 'react-native-add-calendar-event';
+import moment from 'moment';
+import {APOLLOCLIENTADDRESS} from '../../config/constant';
+import {Mutation} from '@apollo/react-components';
+import ApolloClient from 'apollo-boost';
+import gql from 'graphql-tag';
+import {Form} from 'react-final-form';
 
-const Event = props => {
-  return <Text>Event Screen</Text>;
+const COMMENT_MUTATION = gql`
+  mutation comment($username: String!, $comment: String!) {
+    comment(username: $username, comment: $comment) {
+      comments {
+        id
+        username
+        createdAt
+        comment
+      }
+    }
+  }
+`;
 
+const Event = ({
+  title,
+  location,
+  startDate,
+  endDate,
+  description,
+  comments,
+  user,
+}) => {
+  const [value, onChangeText] = React.useState('');
```

Comment:
state should be stored in the container file.

Reply:
Deleted line because form/leaving comments is not part of this merge